week1/vue_1/src/components/TodoCard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div>
+      <input type="checkbox" :checked="todo.done" v-on:change="toggleToDo" v-on:keyup.enter="editToDo"/>
+      <input v-if="editMode" type="text" :value="todo.title" v-on:change="editHandler"/>
```

Comment:
```suggestion
      <input v-if="editMode" type="text" v-model="changedTitle" />
```
이렇게 표현할 수 있겠네요!
대신 created 시점에 changedTitle의 초기 값에 todo.title을 할당해줘야 한답니다
```js
created() {
  this.changedTitle = this.todo.title;
}
```
이렇게요!